src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -77,16 +79,24 @@ def get_data(lookup_key: Union[str, data_keys.SourceTarget], location: str) -> p
         data_keys.ISCHEMIC_STROKE.EMR_CHRONIC: load_emr_ischemic_stroke,
         data_keys.ISCHEMIC_STROKE.CSMR: load_standard_data,
         data_keys.ISCHEMIC_STROKE.RESTRICTIONS: load_metadata,
-        # Cause (myocardial infarction)
-        data_keys.MYOCARDIAL_INFARCTION.PREVALENCE_ACUTE: load_prevalence_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.PREVALENCE_POST: load_prevalence_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.INCIDENCE_RATE_ACUTE: load_incidence_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.DISABILITY_WEIGHT_ACUTE: load_disability_weight_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.DISABILITY_WEIGHT_POST: load_disability_weight_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.EMR_ACUTE: load_emr_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.EMR_POST: load_emr_ihd,
-        data_keys.MYOCARDIAL_INFARCTION.CSMR: load_standard_data,  # Assign 100% of IHD CSMR to MI
-        data_keys.MYOCARDIAL_INFARCTION.RESTRICTIONS: load_metadata,
+        # Cause (ischemic heart disease and heart failure)
+        data_keys.IHD_AND_HF.PREVALENCE_ACUTE_MI: load_prevalence_mi,
+        data_keys.IHD_AND_HF.PREVALENCE_ACUTE_MI_AND_HF: load_prevalence_mi,
+        data_keys.IHD_AND_HF.PREVALENCE_POST_MI: load_prevalence_mi,
+        data_keys.IHD_AND_HF.PREVALENCE_HF_IHD: load_prevalence_heart_failure,
+        data_keys.IHD_AND_HF.PREVALENCE_HF_RESIDUAL: load_prevalence_heart_failure,
+        data_keys.IHD_AND_HF.INCIDENCE_ACUTE_MI: load_incidence_acute_mi,
+        data_keys.IHD_AND_HF.INCIDENCE_HF_IHD: load_incidence_hf_ihd,
+        data_keys.IHD_AND_HF.INCIDENCE_HF_RESIDUAL: load_incidence_hf_residual,
+        data_keys.IHD_AND_HF.DISABILITY_WEIGHT_ACUTE_MI: load_disability_weight_ihd,
+        data_keys.IHD_AND_HF.DISABILITY_WEIGHT_POST_MI: load_disability_weight_ihd,
+        data_keys.IHD_AND_HF.DISABILITY_WEIGHT_HF_IHD: load_disability_weight_hf_ihd,
```

Comment:
I'm not seeing the equations for hf_ihd and residual disability weights in the docs. I assume these come from discussion you have with Syl? Be sure those docs get updated if so

Reply:
I found them [here](https://vivarium-research.readthedocs.io/en/latest/models/causes/ischemic_heart_disease/gbd_2019/index.html#states-data) in the last entry and fourth to last entry for residual heart failure and heart failure from ischemic heart disease respectively.